test/github-issues/904/issue-904.ts

```diff
@@ -0,0 +1,204 @@
+import "reflect-metadata";
+import {createTestingConnections, closeTestingConnections, reloadTestingDatabases} from "../../utils/test-utils";
+import {Connection} from "../../../src/connection/Connection";
+import {Category} from "./entity/Category";
+
+describe("github issues > #904 Using closure tables without @TreeLevelColumn will always fail on insert", () => {
+
+    let connections: Connection[];
+    before(async () => connections = await createTestingConnections({
+        entities: [__dirname + "/entity/*{.js,.ts}"],
+        enabledDrivers: ["mssql", "mysql", "oracle", "postgres", "sqlite"],
```

Comment:
no need to list all drivers if you want it to run for all of them

Reply:
ok